### DISCUSSION TITLE
Scroll down to refresh

### DISCUSSION THREAD

**Discussion by gitbraga** [Telefonica/mistica-design#842] [Proposals]
Hola, podemos empezar un estudio para tener scroll down to refresh y quitar el ícono de refresh?

**Comment by yceballost** [comment]
Buenas Gui!

Pull to refresh tiene muchas implicaciones técnicas en el producto de Novum, lo mejor sería hacer esta petición a Novum en lugar de Mística.

Quizás podría existir una definición visual en Mística pero a la hora de implementarlo en Novum va a ser algo muy diferente que si se implementara en otro producto.

Lo hablo con Ju y Edu para que lo gestionen de la mejor manera :)
